.github/workflows/test.yml

```diff
@@ -0,0 +1,152 @@
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+
+jobs:
+  test-go:
+    runs-on: ubuntu-latest
+    env:
+      LD_API_KEY: ${{ secrets.TEMP_LD_API_TOKEN }}
```

Comment:
`TEMP_`?

Reply:
Was a temp key I made to test things out - will move to a managed key once the repo is imported.